src/backbone.radio.js

```diff
@@ -21,12 +21,11 @@ Radio.DEBUG = false;
 // By default, it logs warning to the console. By overriding this you could
 // make it throw an Error, for instance. This would make firing a nonexistent event
 // have the same consequence as firing a nonexistent method on an Object.
-function debugLog(warning, eventName, channelName) {
-  if (Radio.DEBUG) {
-    var channelText = channelName ? ' on the ' + channelName + ' channel' : '';
-    console.warn(warning + channelText + ': "' + eventName + '"');
-  }
-}
+Radio.debugLog = function(warning, eventName, channelName) {
+  if (!Radio.DEBUG) { return; }
+  var channelText = channelName ? ' on the ' + channelName + ' channel' : '';
+  console.warn(warning + channelText + ': "' + eventName + '"');
```

Comment:
> Non-standard
This feature is non-standard and is not on a standards track. Do not use it on production sites facing the Web: it will not work for every user. There may also be large incompatibilities between implementations and the behavior may change in the future.

https://developer.mozilla.org/en-US/docs/Web/API/Console.warn